content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -155,7 +154,20 @@ spec:
   type: LoadBalancer
 ```
 
-## Garbage Collecting Load Balancers
+### Caveats and limitations when preserving source IPs
+
+Load balancing services from some cloud providers do not let you configure different weights for each target.
+
+With each target weighted equally in terms of sending traffic to Nodes, external traffic is not equally load balanced
+across different Pods. The external load balancer is unaware of the number of Pods on each node that is used as a
```

Comment:
I suspect the last 'is' should be 'are'. If we use 'is', 'that' may refer to 'the number'.